rofeeah_ayeni/alpha_challenges/conditionals.md

```diff
@@ -8,14 +8,30 @@ Chartreuse is my favorite color. Magenta is my second favorite. Periwinkle is my
 ```ruby
 color = "Magenta"
 
-# write your code here
+if color == "Chartreuse"
+  puts "That's my favorite color"
+elsif color == "Magenta"
+  puts "That's my second favorite color"
+elsif color == "Periwinkle"
+  puts "This is my third favorite color"
+else
+  "Yuck! I hate this color."
+end
 ```
 
 Now, turn your color conditional into a method called `color_checker` that takes any color as an argument.
 
 ```ruby
 def color_checker(color)
-  # write your code here
+  if color == "Chartreuse"
+    return "That's my favorite color"
```

Comment:
It's not incorrect to use the explicit `return` here, so no need to change it, but you don't need it - Ruby has an implicit return, which means that it will always return the last evaluated statement 😄